GitLink.py

```diff
@@ -35,13 +35,16 @@
 
 class GitlinkCommand(sublime_plugin.TextCommand):
 
-    def getoutput(self, command):
+    def getoutput(self, command, fallback=None):
         proc = subprocess.Popen(
             command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True,
         )
         out, err = proc.communicate()
         return_code = proc.returncode
         if return_code != 0:
+            if fallback:
+                return fallback
```

Comment:
This will return a fallback value if provided in the event the command above returns a non-zero exit code. I think this is the cleanest way to add default values in here without having to resort to using a catch for control flow around the function call given this function raises a runtime error on non-zero exit codes.